docs/root/development/performance/battery_impact.rst

```diff
@@ -0,0 +1,63 @@
+.. _dev_performance_battery:
+
+Analysis of battery impact
+==========================
+
+In order to identify how Envoy impacts an application, we have created a control application with modifications to our
+current hello world example applications. To see the actual applications we used, you can go to `iOS Envoy example <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/swift/hello_world>`_,
```

Comment:
You should be able to use `:repo:` here instead of full links

Reply:
Does this work for a branch/sha rather than master?

Reply:
I believe so, it's defined here: https://github.com/lyft/envoy-mobile/blob/739ca686e17d2009940c754df15fb232c3494309/docs/conf.py#L72

Reply:
I think that takes in the sha of what the docs are built at. I can just add another method which handles referencing other branches/shas but how about we do that in another PR because we might want to talk about if we want that.

Reply:
Ah, sounds good.